packages/api-server/src/middlewares/jayson.ts

```diff
@@ -11,6 +11,12 @@ export const jaysonMiddleware = (
   res: Response,
   next: NextFunction
 ) => {
+  // we use jayson, and {params: null} will be treated as illegal while undefined will not.
+  // because this line of code https://github.com/tedeh/jayson/blob/master/lib/utils.js#L331
+  if (req.body && req.body.params === null) {
+    req.body.params = undefined;
```

Comment:
Our goal is to allow requests to omit the `params` field when they do not require any. So what about just simply converting the `req.body.params` to empty `unknown[]` or `any[]` here?

```ts
if (req.body && req.body.params == null) {
    req.body.params = [] as any[];
}
```

Reply:
yea, this is over-complex, lets do it in another pr